src/Umbraco.Web.UI.Client/package.json

```diff
@@ -41,6 +41,7 @@
     "spectrum-colorpicker": "1.8.0",
     "tinymce": "4.9.2",
     "typeahead.js": "0.11.1",
+    "typescript": "^3.2.1",
```

Comment:
You have dependencies with fixed versions, why didn't you fix this one?

Reply:
Ah that is no longer needed. I will remove that. Typescript version comes from gulp step